doc/source/code-examples/advancedexamples.rst

```diff
@@ -621,7 +623,10 @@ Here is a simple example using a custom loss function:
     data = np.random.normal(0, 1, size=2**nqubits)
 
     # perform optimization
-    best, params, extra = optimize(myloss, x0, args=(c, data), method='BFGS')
+    optimizer = ScipyMinimizer(options={'method'='BFGS'})
```

Comment:
I would write here a more complete example in which we use both the `options` and the `fit_options` dictionary. I will take care of this.